mail-client/trojita/trojita-0.7.recipe

```diff
@@ -0,0 +1,133 @@
+SUMMARY="A Qt IMAP e-mail client"
+DESCRIPTION="Trojitá enables you to access your mail anytime, anywhere.
+Does not slow you down. If we can improve the productivity of an e-mail user, \
+we better do.
+Respects open standards and facilitate modern technologies. We value the \
+vendor-neutrality that IMAP provides and are committed to be as \
+interoperable as possible.
+Is efficient -- be it at conserving the network bandwidth, keeping memory \
+use at a reasonable level or not hogging the system's CPU.
+Can be used on many platforms. One UI is not enough for everyone, but our \
+IMAP core works fine on anything from desktop computers to cell phones and \
+big ERP systems.
+Plays well with the rest of the ecosystem. We don't like reinventing \
+wheels, but when the existing wheels quite don't fit the tracks, we're \
+not afraid of making them work."
+HOMEPAGE="http://trojita.flaska.net"
+COPYRIGHT="2006-2016 Trojitá developers"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/jktjkt/trojita/archive/v$portVersion.zip"
+CHECKSUM_SHA256="e0e694aabb7dd702e1c4518dd12a17f8124d353c70a293663b4e5c6794129a29"
+ADDITIONAL_FILES="trojita.rdef"
+
+ARCHITECTURES="!x86_gcc2 x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	trojita$secondaryArchSuffix = $portVersion
+	app:Trojita$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku${secondaryArchSuffix}
+	lib:libcrypto$secondaryArchSuffix
+	lib:libqt5$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_PREREQUIRES="
+	cmd:cat
+	cmd:cmake
+	cmd:git
+	cmd:g++$secondaryArchSuffix
+	cmd:make
+#	cmd:mimetic$secondaryArchSuffix
+#	cmd:gpgmepp$secondaryArchSuffix
+	cmd:pkg_config$secondaryArchSuffix
+#	cmd:ragel
+	cmd:sed
+	cmd:python
+	cmd:sh
+	cmd:svn
```

Comment:
Does it build without Subversion?
(If yes, then it would be better to remove the **`cmd:svn`** line from BUILD_PREREQUIRES.)